nx-dev/ui-common/src/lib/discord-icon.tsx

```diff
@@ -0,0 +1,14 @@
+import { FC, SVGProps } from 'react';
+
+export const DiscordIcon: FC<SVGProps<SVGSVGElement>> = (props) => (
```

Comment:
Small nit, but didn't we already have the discord icon somwhere? It's being used in multiple places on the page already

Reply:
Yea, we do but it has always been inlined. I can update those as well but I wanted to keep the change small.